refnx/analysis/__init__.py

```diff
@@ -7,11 +7,10 @@
                                         load_chain)
 from refnx.analysis.model import Model, fitfunc
 
+from refnx._lib._testutils import PytestTester
 
 __all__ = [s for s in dir() if not s.startswith('_')]
 
 
-from refnx._lib._testutils import PytestTester
```

Comment:
why did these need to be moved?

Reply:
PEP8 issues, wants all of the import statements at the top of __init__s